cafebabel/templates/articles/macros.html

```diff
@@ -37,7 +37,7 @@
 
 {%- macro image_upload(object) -%}
   <div class=file>
-    {% if object.image_filename %}
+    {% if object.has_image %}
```

Comment:
It scares me we have something like that here, as for the `get_date` a merge has probably failed somewhere…

Reply:
Mhhh… I manually typed `has_image` here. Do you mean it should remain `image_filename` in this branch?

Reply:
No, I mean we should not have a reference to `image_filename` in the template at first. Which commit added it? There is no mention on `master` AFAIK.

Reply:
Well, Im afraid it's messed up then…
<img width="283" alt="screen shot 2018-02-02 at 2 35 43 pm" src="https://user-images.githubusercontent.com/145172/35753714-6bf09046-0826-11e8-8910-49a63371baed.png">
<img width="233" alt="screen shot 2018-02-02 at 2 35 27 pm" src="https://user-images.githubusercontent.com/145172/35753715-6c01ce60-0826-11e8-893b-32448dbc1edd.png">